homework-submission/react-hyf/react-hyf-client/src/Components/Navbar/Navbar.css

```diff
@@ -0,0 +1,27 @@
+.nav-list {
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0;
+}
+
+.nav-item {
+  border: 0.2px solid rgb(95, 94, 94);
+  padding: 15px;
```

Comment:
I would make the padding 15px on the `nav-link` for better experience: 

![ezgif com-video-to-gif(2)](https://user-images.githubusercontent.com/25272822/61578431-0049ee80-aaf7-11e9-99cf-1d0a243a772b.gif)

Reply:
Many thanks... Done